src/Microsoft.AspNetCore.Mvc.Core/Routing/UrlHelper.cs

```diff
@@ -14,38 +11,18 @@ namespace Microsoft.AspNetCore.Mvc.Routing
     /// An implementation of <see cref="IUrlHelper"/> that contains methods to
     /// build URLs for ASP.NET MVC within an application.
     /// </summary>
-    public class UrlHelper : IUrlHelper
+    public class UrlHelper : UrlHelperBase
```

Comment:
Can we double check that this isn't a .NET breaking change.

Reply:
OK